pkg/inject/inject.go

```diff
@@ -497,6 +501,11 @@ func (conf *ResourceConfig) injectPodSpec(values *patch) {
 	}
 
 	values.AddRootVolumes = len(conf.pod.spec.Volumes) == 0
+
+	if trace := conf.trace(); trace != nil {
+		log.Infof("tracing enabled for %s/%s: remote service=%s, service account=%s", conf.workload.ownerRef.Kind, conf.workload.ownerRef.Name, trace.CollectorSvcAddr, trace.CollectorSvcAccount)
```

Comment:
When running `linkerd inject --manual` against some uninjected payload with the trace annotation, this is throwing a segfault because `conf.workload.ownerRef` is nil.

Reply:
Good catch! Thanks. We got bitten by this before; we can't tell which variables in the config are non-nil, depending on which CLI commands instantiate it.